zeth-contracts/contracts/BaseMixer.sol

```diff
@@ -64,6 +64,10 @@ contract BaseMixer is MerkleTreeMiMC7, ERC223ReceivingContract {
     // If token = address(0) then the mixer works with ether
     address public token;
 
+    // The unit used for public values (ether in and out), in Wei.  Must match
```

Comment:
Please fix the double spacing after each `.` in the files of this PR

Reply:
Replaced.